src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4760,18 +4807,24 @@ namespace SimAirServingZones {
 
 				for ( ZonesCooledNum = 1; ZonesCooledNum <= NumZonesCooled; ++ZonesCooledNum ) { // loop over cooled zones
 					CtrlZoneNum = AirToZoneNodeInfo( AirLoopNum ).CoolCtrlZoneNums( ZonesCooledNum );
-					SysSizing( CurOverallSimDay, AirLoopNum ).NonCoinCoolMassFlow += ZoneSizing( CurOverallSimDay, CtrlZoneNum ).DesCoolMassFlow / ( 1.0 + TermUnitSizing( CtrlZoneNum ).InducRat );
+					int TermUnitSizingIndex = AirToZoneNodeInfo( AirLoopNum ).TermUnitCoolSizingIndex( ZonesCooledNum );
+					Real64 coolFlowRatio = TermUnitSizing( TermUnitSizingIndex ).SpecDesSensCoolingFrac / TermUnitSizing( TermUnitSizingIndex ).SpecDesCoolSATRatio ;
```

Comment:
Potential divide by 0 here and for heatFlowRatio below?

    N2, \field Cooling Design Supply Air Temperature Difference Ratio
      \note This ratio adjusts the supply air temperature difference used to calculate 
      \note the cooling design supply air flow rate for this terminal unit.
      \type real
      \units dimensionless
      \minimum 0.0